shared/src/validation-schemas/validation-schemas.ts

```diff
@@ -6,6 +6,7 @@ export {
   courseUpdateCategory,
 } from './course/course';
 export { courseCategoryGetByIdParams } from './course-categories/course-categories';
+export { courseMentorCreateBody } from './course-mentor/course-mentor';
```

Comment:
```suggestion
export { courseMentorCreate } from './course-mentor/course-mentor';
```
don't forget about filename